### ISSUE TITLE
[Accessibility] Accessibility requirements should be indicated per-control

### ISSUE THREAD

**Issue by citelao** [microsoft/WinUI-Gallery#1230]
### Which version of the app?

WinUI 3 Gallery

### Description

Certain controls always have specific accessibility requirements—for example, `TextBox`es and `Image`s (and typically `*View`s) always need an accessible name.

We should be explicit and add that information to the examples for each page. For example, we should indicate that requirement on the `TextBox` page and the `Image` page (not just the Screen reader page).

**Comment by niels9001** [comment]
The Gallery is focused very much on how to use certain controls. We have extensive a11y guidance in the docs on MS Learn.
